src/helper/emailServices.js

```
@@ -0,0 +1,91 @@
+const nodemailer = require("nodemailer");
+const { compile } = require("handlebars");
+const mjml2html = require("mjml");
+const path = require("path");
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)